### ISSUE TITLE
Strange Red Icon

### ISSUE THREAD

**Issue by FireWings2259** [3axap4eHko/OWA-Notifier#54]
A small white charier is constantly showing in the bottom right hand corner of the app. Don't know if this is a bug or not but I think it is as it is there then its not and it just stays there.

Thanks

**Comment by 3axap4eHko** [comment]
Do you have a screen?

**Comment by j-martyn** [comment]
I have the same problem.
![image](https://cloud.githubusercontent.com/assets/2838066/9584681/2e0d24bc-501b-11e5-98b0-068e59a4fdc4.png)

**Comment by j-martyn** [comment]
Sometimes this character is disappearing.

There is also problem with with blinking mail icon, maybe there is connection between them.

![2015-08-31 20_07_04-owa-notifier_popup js at 0440b074b7d18f125aeb9c650bbd14f080b5aa4f 3axap4ehko_o](https://cloud.githubusercontent.com/assets/2838066/9585021/ee744612-501c-11e5-879a-027fb1474314.png)

**Comment by j-martyn** [comment]
[`/src/js/Extension.js:163`](https://github.com/3axap4eHko/OWA-Notifier/blob/0440b074b7d18f125aeb9c650bbd14f080b5aa4f/src/js/Extension.js#L163) - here is where  the character from
```javascript
        if (globalWarning) {
            badgeText += ' !';
        }
```
It seems to be appearing after bad 500 response for request
```http
POST /ews/Exchange.asmx HTTP/1.1
```
with request payload
```xml
<?xml version="1.0" encoding="utf-8"?>
<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xmlns:m="http://schemas.microsoft.com/exchange/services/2006/messages"
               xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types"
               xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
    <soap:Body>
        <m:FindItem Traversal="Shallow">
            <m:ItemShape>
                <t:BaseShape>IdOnly</t:BaseShape>
                <t:AdditionalProperties>
                    <t:FieldURI FieldURI="item:Subject" />
                    <t:FieldURI FieldURI="item:ReminderMinutesBeforeStart" />
                    <t:FieldURI FieldURI="calendar:Start" />
                    <t:FieldURI FieldURI="calendar:End" />
                    <t:FieldURI FieldURI="calendar:Location" />
                </t:AdditionalProperties>
            </m:ItemShape>
            <m:CalendarView MaxEntriesReturned="10" StartDate="2015-08-31T20:24:28--03:00" EndDate="2015-09-10T20:24:28--03:00" />
            <m:ParentFolderIds>
                <t:DistinguishedFolderId Id="calendar"/>
            </m:ParentFolderIds>
        </m:FindItem>
    </soap:Body>
</soap:Envelope>
```
and response 
```http
HTTP/1.1 500 Internal Server Error
....
```
with body
```xml
<?xml version="1.0" encoding="utf-8"?>
<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/">
    <s:Body>
        <s:Fault>
            <faultcode xmlns:a="http://schemas.microsoft.com/exchange/services/2006/types">a:ErrorSchemaValidation
            </faultcode>
            <faultstring xml:lang="uk-UA">Не удалось выполнить проверку запроса на соответствие схеме: Атрибут
                "StartDate" недействителен: значение "2015-08-31T20:28:58--03:00" недействительно с зрения его типа
                данных "http://www.w3.org/2001/XMLSchema:dateTime" — Строка "2015-08-31T20:28:58--03:00" не является
                допустимым значением XsdDateTime.
            </faultstring>
            <detail>
                <e:ResponseCode xmlns:e="http://schemas.microsoft.com/exchange/services/2006/errors">
                    ErrorSchemaValidation
                </e:ResponseCode>
                <e:Message xmlns:e="http://schemas.microsoft.com/exchange/services/2006/errors">Запрос не прошел
                    проверку на соответствие схеме.
                </e:Message>
                <t:MessageXml xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types">
                    <t:LineNumber>18</t:LineNumber>
                    <t:LinePosition>53</t:LinePosition>
                    <t:Violation>Атрибут "StartDate" недействителен: значение "2015-08-31T20:28:58--03:00"
                        недействительно с зрения его типа данных "http://www.w3.org/2001/XMLSchema:dateTime" — Строка
                        "2015-08-31T20:28:58--03:00" не является допустимым значением XsdDateTime.
                    </t:Violation>
                </t:MessageXml>
            </detail>
        </s:Fault>
    </s:Body>
</s:Envelope>
```

**Comment by 3axap4eHko** [comment]
Oh! Thank you! This will help me a lot!

**Comment by 3axap4eHko** [comment]
Fixed in 3.0.2.4
